### DISCUSSION TITLE
Calculate Distance Camera from the object

### DISCUSSION THREAD

**Discussion by Hafikan** [kornia/kornia#1383] [General]
Hello, firstly sorry for my English. I preparing for International Teknofest unmanned aerial vehicle competition. I have a big problem. I use OpenCv for image processing and i have one Camera. We can not find distance Camera to object on openCv. Can we calculate distance with only one camera on Kornia

**Comment by edgarriba** [comment]
@Hafikan this will be possible after merging `solve_pnp_dlt` in https://github.com/kornia/kornia/pull/1349. /cc @thatbrguy

**Reply by thatbrguy** [reply]
Well, if the user knows 3D-2D correspondences of the object in world space and image space, then the distance of those 3D object points to the camera can be calculated with some additional code after using `solve_pnp_dlt` to get the world to camera matrix. 

But, if the correspondences are not known, then `solve_pnp_dlt` won't be useful right? The user may have to use some other strategy then.
